src/utils/getComposerWorkingDir.ts

```diff
@@ -0,0 +1,19 @@
+import {info} from '@actions/core'
+import {parseStringToArguments} from './parseStringToArguments'
+
+export function getComposerWorkingDir(composerOptions = ''): string {
+  const composerOptionsParsed = parseStringToArguments(composerOptions)
+  const composerWorkingDir = (
+    (composerOptionsParsed['working-dir'] as string) ||
+    (composerOptionsParsed.d as string) ||
+    ''
+  ).trim()
+
+  if (composerWorkingDir === '') {
+    info(`Composer working directory found at project root.`)
+  } else {
+    info(`Composer working directory found at "${composerWorkingDir}".`)
```

Comment:
Since this isn't actually confirming the existence of the Composer working directory (i.e., we haven't *found* it because we didn't look for it), let's change "found at" to "set to."

```suggestion
    info(`Composer working directory set to project root.`)
  } else {
    info(`Composer working directory set to "${composerWorkingDir}".`)
```